net/unbound/files/unbound.hotplug

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+[ "$ACTION" = ifup ] || exit 0
+/etc/init.d/unbound enabled || exit 0
+/etc/init.d/unbound restart
```

Comment:
IFUP needs to be evaluated. This is necessary for generating the PTR records to forward to dnsmasq (UCI dnsmsaq), and to select interfaces that Unbound will listen on exclusively (UCI local_service).  I can make it a separate commit, but the overall design doesn't work without it.